datasets/ru/dossier_center_pep/ru_dosseier_center_pep.yml

```diff
@@ -0,0 +1,21 @@
+name: ru_dossier_center_pep
```

Comment:
Let's add [load_db_uri ](https://zavod.opensanctions.org/metadata/#exports) - a nice effect this will have I think is that the notes will show up on their profiles on OpenSanctions